prog/prio.go

```diff
@@ -137,7 +137,11 @@ type weights struct {
 	inout int32
 }
 
-func noteUsage(uses map[string]map[int]weights, c *Syscall, weight int32, dir Dir, str string, args ...interface{}) {
+func noteUsage(uses map[string]map[int]weights, c *Syscall, weight int32, dir Dir, str string) {
```

Comment:
Does the linter warn about this as well? How does it even know this is related to string formatting?....

Reply:
Yes. It warned about the noteUsage.